src/components/TicketDiv.tsx

```diff
@@ -0,0 +1,43 @@
+import { CSSProperties } from 'react';
+
+type Props = {
+  img: string
+  alt: string
+  headline: string
+  price: string
+  capacity: string
+  info: string
+}
+
+const infoStyle: CSSProperties = {
+  maxWidth: '450px',
+  maxHeight: '50px', 
+  overflowY: 'auto', 
+  padding: '0 10px', 
+};
+
+const imgStyle: CSSProperties = {
```

Comment:
same with these, just be aware, there's only one box shadow class that should be enough for all browsers